Microsoft.Toolkit.Services/AssemblyInfo.cs

```diff
@@ -3,15 +3,5 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Runtime.CompilerServices;
-[assembly:InternalsVisibleTo("Microsoft.Toolkit.Uwp.Services, PublicKey=00240000048000009400000006020000002" +
```

Comment:
Not sure what internal types UWP services is using, but you can't have a strong named assm expose types to a non-strong named one.